lib/checkout.rb

```diff
@@ -0,0 +1,24 @@
+# Download the helper library from https://www.twilio.com/docs/ruby/install
+require 'twilio-ruby'
+
+# Find your Account SID and Auth Token at twilio.com/console
+# and set the environment variables. See http://twil.io/secure
+
+class Checkout
+  def initialize(_client = Twilio::REST::Client, _sid = ENV['account_sid'], _auth = ENV['auth_token'])
```

Comment:
I didn't think to inject these as arguments to my Message class. I'm curious as to why you decided to do this, as opposed to just using them directly as variables within the message method? It certainly makes your code clearer, but I'm curious if you had another reason, other than readability, for doing it this way :) (asking for the purposes of my own learning as I'm trying to get to grips with dependency injection so any insight into others' approaches is interesting!)